src/styles/_PopUp.scss

```diff
@@ -92,9 +93,35 @@
 
 .popup-image {
   height: 200px;
+  padding-top: 5%;
 }
 
 .youtube-video {
   object-fit: contain;
   height: 500px;
+}
+
+@media screen and (max-width: 1030px) {
+  .age-range, 
+  .num-of-players, 
+  .num-of-minutes,
+  .game-info,
+  .difficulty-level,
+  .game-description {
+    font-size: 12px;
+  }
+
+  .game-name {
+    font-size: 30px;
+  }
+}
+
+@media screen and (max-width: 770px) {
+  .PopUp {
+    grid-template-columns: 1fr;
+  }
+
+  .youtube-video {
+    display: none;
+  }
```

Comment:
I think this should be 880px to match the change in SearchPage.scss that went from 770 to 880. Video goes away at 880 then card expands at 770

Reply:
Definitely, good catch! Will do.